content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

```diff
@@ -240,17 +238,16 @@ TLS origination.
 1.  Change directory to the cloned repository:
 
     {{< text bash >}}
-    $ cd mtls-go-example
+    $ cd mtls-go-example || exit
```

Comment:
I don't think we should change the docs in ways that will make no sense for users that are following the task and just running these commands from a terminal. I think this is a good example of a lint error that is not applicable and we should add it to the list of disabled ones in the generator: https://github.com/istio/istio.io/blob/78c6269fe7ad7fe0659366bdf0ec3f8197a2c030/scripts/snip.py#L32.

You should probably make this change in a separate PR because it will require a regen of every current snip.sh, so we should make the change, regen, and then merge, without delay.

Reply:
```
# shellcheck disable=SC2034,SC2153,SC2155,SC2164
```

Reply:
done